src/applications/static-pages/facilities/vet-center/NearByVetCenters.jsx

```diff
@@ -222,15 +225,31 @@ const NearByVetCenters = props => {
     );
     return renderNearbyVetCenterContainer(normalizedFetchedVetCenters);
   }
+  if (originalCoordinates && fetchedVetCenters.length > 0) {
+    return (
+      <>
+        <h2
+          className="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 vads-u-margin-bottom--2p5
+                  medium-screen:vads-u-margin-bottom--3"
+          id="other-near-locations-not-found"
+          data-testid="other-near-locations-not-found"
+        >
+          Other nearby Vet Centers
+        </h2>
+        <p>No nearby Vet Center locations found.</p>
```

Comment:
Do we need to show this section at all if nearby vet center locations are not found? I'm wondering whether the user would expect to see this section and be confused if it were missing (even if there weren't data for it).

Reply:
We had this discussion around VBA and the decision was that we should show it there. The consensus from Jill and Dave and others was to keep it here as well.

Reply:
Gotcha. Thanks!